lib/assessment.md

```diff
@@ -163,5 +210,5 @@ Consider the following media queries. How wide would the screen need to be for e
 ```
 
 <!-- your answer starts here -->
-
+700px
```

Comment:
Close, but not correct. At a screen width of 800px, which media queries have 'visible' CSS code? Given that, what does the CSS say the width of `.stretch` elements should be?